app/application/usecase/account/login_user.go

```diff
@@ -0,0 +1,62 @@
+package account
+
+import (
+	"context"
+	"os"
+	"stoneBanking/app/application/validations"
+	"stoneBanking/app/application/vo/input"
+	"stoneBanking/app/application/vo/output"
+	"stoneBanking/app/common/utils"
+	"stoneBanking/app/domain/entities/account"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+type ClaimStruct struct {
+	jwt.StandardClaims
+	User_id int
+}
+
+func (usecase *usecase) LoginUser(loginInput input.LoginVO) (output.LoginOutputVO, error) {
```

Comment:
Normalmente passamos o context.Context como primeiro parametro de todos os usecase. Isso se deve ao fato do context reter alguns valores como "request-id" que serve para rastrear o que cada request HTTP faz no nosso sistema e também para guardar a transação do newrelic e com isso obtermos métricas de latência, uso de CPU, erros para cada request.

Acredito que esse detalhamento não deve entrar no seu desafio, mas fica ai a sugestão de estudo do uso de context para essas coisas :)

Um link para você dar uma olhada [Using context](https://www.sohamkamani.com/golang/context-cancellation-and-values/)

Reply:
Obrigado, vou dar uma lida e me aprofundar nisso. Entendendo, eu volto e melhoro o código.